dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'snapchat_ads_source'
-version: '0.3.1'
+version: '0.4.0'
```

Comment:
Can we change the custom schema on line 18 in this file to be `snapchat_ads_source` to be consistent with our other packages.